modular_skyrat/modules/modular_weapons/code/company_and_or_faction_based/szot_dynamica/rifle.dm

```diff
@@ -0,0 +1,158 @@
+// Semi-automatic rifle firing .310 with reduced damage compared to a Sakhno
+
+/obj/item/gun/ballistic/automatic/lanca
+	name = "\improper Szot 'Lanca' Battle Rifle"
+	desc = "A relatively compact, long barreled bullpup battle rifle chambered for .310 Strilka. Has an integrated sight with \
+		a surprisingly functional amount of magnification, given its place of origin."
+
+	icon = 'modular_skyrat/modules/modular_weapons/icons/obj/company_and_or_faction_based/szot_dynamica/guns_48.dmi'
+	icon_state = "lanca"
+
+	worn_icon = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/szot_dynamica/guns_worn.dmi'
+	worn_icon_state = "lanca"
+
+	lefthand_file = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/szot_dynamica/guns_lefthand.dmi'
+	righthand_file = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/szot_dynamica/guns_righthand.dmi'
+	inhand_icon_state = "lanca"
+
+	SET_BASE_PIXEL(-8, 0)
+
+	special_mags = FALSE
+
+	bolt_type = BOLT_TYPE_STANDARD
+
+	w_class = WEIGHT_CLASS_BULKY
+	weapon_weight = WEAPON_HEAVY
+	slot_flags = ITEM_SLOT_BACK | ITEM_SLOT_OCLOTHING
+
+	accepted_magazine_type = /obj/item/ammo_box/magazine/lanca
+
+	fire_sound = 'modular_skyrat/modules/modular_weapons/sounds/battle_rifle.ogg'
+	suppressed_sound = 'modular_skyrat/modules/modular_weapons/sounds/suppressed_heavy.ogg'
+	can_suppress = FALSE
+
+	can_bayonet = FALSE
+
+	burst_size = 1
+	fire_delay = 0.6 SECONDS
+	actions_types = list()
+
+	recoil = 0.25
+	spread = 2.5
+	projectile_wound_bonus = -10
+	projectile_damage_multiplier = 0.85
```

Comment:
Again, as I said in the past, I'd like if we tried to avoid using `projectile_damage_multiplier` as much as possible, because it's something that's not readily apparent to players, and makes choosing weapons and ammo more complicated for nothing.